R/RLearner_classif_glmboost.R

```diff
@@ -49,12 +54,17 @@ trainLearner.classif.glmboost = function(.learner, .task, .subset, .weights = NU
 predictLearner.classif.glmboost = function(.learner, .model, .newdata, ...) {
   type = ifelse(.learner$predict.type == "response", "class", "response")
   p = predict(.model$learner.model, newdata = .newdata, type = type, ...)
+  fam = getLearnerParVals(.learner)$family
   if (.learner$predict.type  == "prob") {
-    td = .model$task.desc
-    p = p[, 1L]
-    levs = c(td$negative, td$positive)
-    return(propVectorToMatrix(p, levs))
+    if (fam %in% c("AdaExp", "AUC")) {
+      stopf("prediction.type = 'prob' not implemented for family %s", fam)
+    } else {
+      td = .model$task.desc
```

Comment:
What is going on in this else block, please add a short comment what is done here